src/core/etl/src/Flow/ETL/Extractor/Signal.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
+
+namespace Flow\ETL\Extractor;
+
+enum Signal
+{
+    case STOP;
```

Comment:
What about `STOP_PROCESSING`? Cause stop is quite a generic name

Reply:
Extractors are not really processing, they are extracting, but I didn't want yet to implement something super specific, I want to wait and see how this will evolve, the moment STOP will become too enigmatic, then we might need to think about splitting it